CHANGELOG.md

```diff
@@ -10,6 +10,8 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
 ### 🛡 Security
 
+- [CVE-2023-45857] Bump `axios` from `0.27.2` to `0.28.0` ([#5470](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/5470))
```

Comment:
Do we want to update the PR number here? or separate changelog for just the axios?